applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -204,7 +204,7 @@ struct cloud_data_neighbor_cells {
 	/** Contains current cell and number of neighbor cells. */
 	struct lte_lc_cells_info cell_data;
 	/** Contains neighborhood cells. */
-	struct lte_lc_ncell neighbor_cells[17];
+	struct lte_lc_ncell neighbor_cells[CONFIG_LTE_NEIGHBOR_CELLS_MAX];
```

Comment:
I need to revert this because iot_thingy91 tests have qemu tests that don't understand `CONFIG_LTE_NEIGHBOR_CELLS_MAX`. I think tests should be modified to define `CONFIG_LTE_NEIGHBOR_CELLS_MAX` but we don't have time for that now.